File: examples/documents/model-delete.swift
import JsBaoClient

// Delete a record: load it, then delete.
func deleteTask(documentId: String, taskId: String) async throws {
  // #region example
  if let task = try await Task.find(taskId) {
    try task.delete(in: documentId)
  }
  // #endregion example
}

File: examples/documents/model-delete.ts
import { Task } from "../_harness/generated/ts/Task.generated";

// Delete a record: load it, then delete.
export async function deleteTask(taskId: string) {
  // #region example
  const task = await Task.find(taskId);
  if (task) {
    await task.delete();
  }
  // #endregion example
}
